src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+import { getPeople } from "../../api/people";
+import { v4 as uuidv4 } from 'uuid';
+import { PeopleTable } from "../PeopleTable/PeopleTable";
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<PersonWithParents[]>([]);
+
+  useEffect(() => {
+    getPeople()
+      .then(requestPeople => {
+
+        const newPeople: Array<PersonWithParents> = requestPeople.map((person: Person) => {
+          const father: Person | undefined = requestPeople.find(({ name }: Person) => name === person.fatherName);
+          const mother: Person | undefined = requestPeople.find(({ name }: Person) => name === person.motherName);
```

Comment:
```suggestion
          const father: Person = requestPeople.find(({ name }: Person) => name === person.fatherName);
          const mother: Person = requestPeople.find(({ name }: Person) => name === person.motherName);
```
Find вроде сам должен типизировать это как возможный undefined

Reply:
Да, так і є)